hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -21,7 +21,7 @@ import org.apache.spark.sql.Row
 
 object ExecStrategy extends Enumeration {
   type ExecStrategy = Value
-  val Interpret, InterpretUnoptimized, JvmCompile, LoweredJVMCompile = Value
+  val Interpret, InterpretUnoptimized, JvmCompile, LoweredJVMCompile, JvmCompileUnoptimized = Value
```

Comment:
we should add JVMCompileUnoptimized to the sets below, but let's do that in a separate PR -- have a feeling things are going to fail 😉

Reply:
They definitely do, 2 failures at least, in one case a literal of an empty array, in the other (testCoalesce) possibly something similar